webui/react/src/hooks/useModal/Model/useModalModelDelete.tsx

```diff
@@ -55,7 +55,7 @@ const useModalModelDelete = ({ onClose }: Props = {}): ModalHooks => {
       onOk: handleOk,
       title: 'Confirm Delete',
     } : clone(CANNOT_DELETE_MODAL_PROPS);
-  }, [ user?.id, user?.isAdmin ]);
+  }, [ canDeleteModel ]);
```

Comment:
I think it's okay to add a `permissions.rawPermissions`that returns an array of permissions that can be used in places like this.

Reply:
There's some difficulty because in RBAC we just need the `delete_model` permission, but in OSS the permissions here depends on context of who owns the model (+ future Groups logic)

Reply:
For OSS we can just create a client-side only oss_admin permission object that we return in `rawPermissions`that would preserve the old behavior here. Though do we ever have to actually invalid this callback? When are users becoming admins during interaction?